### DISCUSSION TITLE
--privileged on podman, will it possible to incorporate it into podman-composer? Is there such same thing docker in docker?

### DISCUSSION THREAD

**Discussion by benyaminl** [containers/podman-compose#619] [General]
It's related to https://www.redhat.com/sysadmin/podman-inside-container by @rhatdan, but I really want to know, does docker in docker, need to have special config on docker-compose.yml or it's not? 

Because on podman, when we need podman in podman, we need --privileged in it, and I think it's hard to replicate in docker-compose.yml or podman-compose.yml or container-compose.yml

If I have wrong concept would anyone please help me to understand, which one is wrong, and what I need to know more. Thank you

**Comment by Konstanty** [comment]
If you want to use `docker-compose.yml` and get the --privileged flag then just add this to the definition:

```
   privileged: true
```

**Reply by benyaminl** [reply]
I will try that. Thanks
